tox.ini

```diff
@@ -39,13 +39,7 @@ commands =
   flake8
 
 [testenv:pylint]
-deps =
-  aiocoap
-  iotlabcli
-  pylint
-  pytest
-  riotctrl
-  scapy
+deps = requirements.txt
```

Comment:
Wouldn't it make sense (even more sense) to also add this to the `pytest` `testenv`?

Reply:
Ah yes sorry.

Reply:
Hmm did not work